### DISCUSSION TITLE
Samsung smartthings dashboard

### DISCUSSION THREAD

**Discussion by cmoneto** [BimmerGestalt/HassGestalt#17] [Ideas]
Hi, great app!  Is there a way to use this with samsung smartthing? 🫣

**Comment by hufman** [comment]
Good afternoon!
HassGestalt itself can only talk to Home Assistant. Home Assistant has a [plugin to control Samsung SmartThings](https://www.home-assistant.io/integrations/smartthings/) which would let you use HassGestalt with SmartThings, but there's no direct method otherwise.
